app/build.gradle

```diff
@@ -48,6 +49,10 @@ configurations {
     implementation.exclude module: 'com.android.tools.external.lombok'
 }
 
+repositories {
+    jcenter()
```

Comment:
I'm not sure (And I can't check right now). Can you please verify `jcenter` it's not included by default? (Try removing this and compiling without). It it is, please remove, else just comment and close this discussion